src/main/java/org/opensearch/security/hasher/PasswordHasher.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.hasher;
+
+public interface PasswordHasher {
```

Comment:
is there a plan to implement multiple Password Hashers?

Reply:
Yes, in the upcoming PR's!